fw/http_limits.c

```diff
@@ -564,6 +564,49 @@ frang_http_methods_override(const TfwHttpReq *req, FrangAcc *ra,
 	return TFW_PASS;
 }
 
+static int
+frang_http_upgrade_websocket(const TfwHttpReq *req, FrangAcc *ra,
+			     FrangVhostCfg *f_cfg)
+{
+	BUG_ON(!req);
+
+	switch (req->version) {
+	/*
+	 * TODO upgrade websocket checks for h2 as described in RFC8441
```

Comment:
Usually we add the issue number for TODO comments and when we start on an issue it's handy to grep the code for something like `TODO #775`